addons/libs/packets/fields.lua

```diff
@@ -2098,6 +2098,8 @@ types.shop_item = L{
     {ctype='unsigned int',      label='Price',              fn=gil},            -- 00
     {ctype='unsigned short',    label='Item',               fn=item},           -- 04
     {ctype='unsigned short',    label='Shop Slot'},                             -- 08
+	{ctype='unsigned short',    label='Craft Skill'}                            -- Shows up on Guild NPCs.  Unknown.
+	{ctype='unsigned short',    label='Craft Rank'},                            -- Correlates to Rank able to purchase product from GuildNPC
```

Comment:
Whatever editor you're using, find out how to change tabs to spaces, make 1 tab = 4 spaces. :)